lib/instance.js

```diff
@@ -28,7 +28,12 @@ function Instance(options) {
   this.process = child.spawn(this.command, this.args, {
     detached: options.detached,
     env: options.env,
-    cwd: options.cwd
+    cwd: options.cwd,
+    stdio: ['ignore', 1, 2]
+  });
+
+  this.process.on('data', function (data) {
+    console.log(data.toString());
```

Comment:
Don't these two changes (Ignoring `stdout` and `stderr` and then printing out the processes output) cancel each other out?